### PR TITLE
MCFM+JHUGen background input cards 13TeV pdf 

### PR DESCRIPTION
**Pull request by carolhungwt**

### CODE REVIEW

File: bin/Powheg/submitjobs.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```
Comment (perrozzi) [inline bin/Powheg/submitjobs.py]: Hi, do we really need another layer? probably this can stay as private script...

File: bin/Powheg/run_pwg.py [hunk 2]
```diff
@@ -1174,17 +1175,16 @@ def runhnnlo(folderName, njobs, QUEUE):
                     test_pdf2 = n_column[1].strip()
 
             if not (test_pdf1 == test_pdf2) :
-                print "ERROR: PDF settings not equal for the 2 protons: "+test_pdf1+" vs "+test_pdf2+"... Please check your datacard"
-                quit()
+                raise RuntimeError("ERROR: PDF settings not equal for the 2 protons: {0} vs {1}... Please check your datacard".format(test_pdf1, test_pdf2))
```
Comment (perrozzi) [inline bin/Powheg/run_pwg.py]: you have tested this, right? it saves one line but does exactly the same thing I suppose

File: bin/MCFM/runcmsgrid_template.sh [hunk 3]
```diff
@@ -44,7 +44,7 @@ cd $LHEWORKDIR
 
 basename=BASENAME
 
-cd $LHEWORKDIR/$basename
+#cd $LHEWORKDIR/$basename
```
Comment (perrozzi) [inline bin/MCFM/runcmsgrid_template.sh]: can you maybe delete the commented lines?

File: bin/Powheg/run_pwg.py (hunk 2 above)
Comment (carolhungwt) [reply bin/Powheg/run_pwg.py]: Yes, this has been tested in recent gridpack production

File: bin/Powheg/submitjobs.py [hunk 4]
```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/env python
```
Comment (perrozzi) [inline bin/Powheg/submitjobs.py]: I don't see this commit doing what is written, can you please check?

File: bin/MCFM/cards/MCFM+JHUGen/ggZZBKG.DAT [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+[Heavy quark masses]
+173.2d0		[top mass]
+4.75d0		[bottom mass]
+1.275d0		[charm mass]
+
+[Pdf selection]
+cteq6_l [pdlabel]
+4       [NGROUP, see PDFLIB]
+46      [NSET - see PDFLIB]
+'NNPDF30_lo_as_0131'  [LHAPDF group]
```
Comment (perrozzi) [inline bin/MCFM/cards/MCFM+JHUGen/ggZZBKG.DAT]: is there any specific reason to keep NNPDF30 in some cases, instead of NNPDF31?

Comment (carolhungwt) [reply bin/MCFM/cards/MCFM+JHUGen/ggZZBKG.DAT]: Sorry, i should better remove this, this is a template input card

File: bin/Powheg/production/2017/14TeV/ZZ/ZZ_4L_NNPDF31_14TeV_pdf260000.input [hunk 6]
```diff
@@ -0,0 +1,37 @@
+! ZZ boson production parameters
+mllmin 4d0         ! default 0.1 GeV this is minimum invar mass for Z leptons
+
+numevts NEVENTS   ! number of events to be generated
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 7000d0     ! energy of beam 1
+ebeam2 7000d0     ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1   260000   ! pdf set for hadron 1 (LHA numbering)
```
Comment (perrozzi) [inline bin/Powheg/production/2017/14TeV/ZZ/ZZ_4L_NNPDF31_14TeV_pdf260000.input]: this looks like a pre2017 datacard, why is in the 2017 folder?

File: bin/MCFM/cards/MCFM+JHUGen/MCFM_JHUGen_13TeV_ggZZtoMUTL_BKG.DAT [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+[Heavy quark masses]
+173.2d0		[top mass]
+4.75d0		[bottom mass]
+1.275d0		[charm mass]
+
+[Pdf selection]
+cteq6_l [pdlabel]
+4       [NGROUP, see PDFLIB]
+46      [NSET - see PDFLIB]
+'NNPDF31_lo_as_0131'			[LHAPDF group]
```
Comment (perrozzi) [inline bin/MCFM/cards/MCFM+JHUGen/MCFM_JHUGen_13TeV_ggZZtoMUTL_BKG.DAT]: would it make sense to include the pdf set (either NNPDF3 or the lhaid number) in the filename as for the powheg datacards?

**Pull request merged**